scripts/build-motion-library-v5.py

```
@@ -185,6 +185,55 @@ def vnorm(v):
+    """World +Z basis rotated by quat q=(x,y,z,w) → world forward vector."""
+    x, y, z, w = q
+    return (2 * (x * z + w * y), 2 * (y * z - w * x), 1 - 2 * (x * x + y * y))
+
+
+def _mean_hip_yaw_deg(quats):
+    """Mean absolute yaw (deg off +Z) of the hip's world forward across the
+    clip. ~0 = faces straight ahead; large = the stride is baked sideways
+    (the stylized/quadruped-rig locomotion failure)."""
+    ys = []
+    for f in quats:
+        fx, fy, fz = _quat_fwd(f[HIP])
```

Ruff (0.15.21)

[warning] Unpacked variable `fy` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)